com.unity.robotics.urdf-importer/Runtime/MeshProcessing/StlAssetPostProcessor.cs

```diff
@@ -28,59 +28,66 @@ public class StlAssetPostProcessor
         {
+            const bool enableStlPostprocessing = false; // AIRO-908 
 #if UNITY_EDITOR
-            if (!RuntimeURDF.IsRuntimeMode())
+            if (!RuntimeURDF.IsRuntimeMode() && enableStlPostprocessing)
             {
                 foreach (var stlFile in importedAssets.Where(x => x.ToLowerInvariant().EndsWith(".stl")))
                 {
-                    var stlFileLowercase = stlFile.ToLower();
-                    if (stlFileLowercase.StartsWith("assets"))
-                    {
-                        Debug.Log($"Detected an stl file at {stlFile} - creating a mesh prefab.");
-                        createStlPrefab(stlFile);
-                    }
-                    else if (stlFileLowercase.StartsWith("packages"))
-                    {
-                        Debug.Log($"Found an stl file at {stlFile} - " + 
-                            "skipping post-processing because it's a Package asset");
-                    }
-                    else
-                    {
-                        Debug.LogWarning($"Found an stl file at {stlFile} - " + 
-                            "skipping post-processing because we don't know how to handle an asset in this location.");
-                    }
+                    PostprocessStlFile(stlFile);
                 }
             }
 #endif
         }
 
-        private static void createStlPrefab(string stlFile)
+        public static void PostprocessStlFile(string stlFile)
+        {
+            var stlFileLowercase = stlFile.ToLower();
+            if (stlFileLowercase.StartsWith("assets"))
+            {
+                Debug.Log($"Detected an stl file at {stlFile} - creating a mesh prefab.");
+                CreateStlPrefab(stlFile);
+            }
+            else if (stlFileLowercase.StartsWith("packages"))
+            {
+                Debug.Log($"Found an stl file at {stlFile} - " + 
+                          "skipping post-processing because it's a Package asset");
+            }
+            else
+            {
+                Debug.LogWarning($"Found an stl file at {stlFile} - " + 
+                                 "skipping post-processing because we don't know how to handle an asset in this location.");
+            }
+        }
+
+        private static void CreateStlPrefab(string stlFile)
         {
             GameObject gameObject = CreateStlParent(stlFile);
-            if (gameObject == null)
+            if (!gameObject)
             {
+                Debug.LogWarning($"Could not create a mesh prefab for {stlFile}");
                 return;
             }
 
-            RuntimeURDF.PrefabUtility_SaveAsPrefabAsset(gameObject, getPrefabAssetPath(stlFile));
+            RuntimeURDF.PrefabUtility_SaveAsPrefabAsset(gameObject, GetPrefabAssetPath(stlFile));
             Object.DestroyImmediate(gameObject);
         }
 
-        private static Material defaultDiffuse = null;
+        private static Material s_DefaultDiffuse = null;
```

Comment:
Could this go to the top of the class with the rest of the member declarations?

Reply:
done